packages/component-utilities/src/echarts.js

```diff
@@ -424,6 +424,17 @@ export default (node, option) => {
 
 	const chart = init(node, 'evidence-light', { renderer: 'svg' });
 
+	// If the x-axis of a series is numeric, or a date; ensure that it is in order
+	option.series = option.series.map((s) => {
+		if (typeof s.data[0][0] === 'number') {
+			s.data = s.data.sort((a, b) => a[0] - b[0]);
+		} else if (s.data[0][0] instanceof Date) {
```

Comment:
```suggestion
		if (typeof s.data?.[0][0] === 'number') {
			s.data = s.data.sort((a, b) => a[0] - b[0]);
		} else if (s.data?.[0][0] instanceof Date) {
```
Seems like `/charts/bar-chart` from `example-project` errors out w/o this